src/interfaces/IERC3009.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.23;
+
+interface IERC3009 {
```

Comment:
If `ERC3009 is IERC3009, StatefulERC712`, then `IERC3009 is IStatefulERC712`.

Reply:
Fixed in this commit: https://github.com/MZero-Labs/common/pull/7/commits/287b68343734e04236e9d631ee9a4d0e8db50a21